src/SignalSlot/RemoveTranslationSlot.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformHttpCacheBundle\SignalSlot;
+
+use EzSystems\PlatformHttpCacheBundle\PurgeClient\PurgeClientInterface;
+use eZ\Publish\Core\SignalSlot\Signal;
+use eZ\Publish\SPI\Persistence\Handler as PersistenceHandler;
+
+class RemoveTranslationSlot extends AbstractContentSlot
+{
+    /**
+     * @var \eZ\Publish\SPI\Persistence\Handler
+     */
+    protected $persistenceHandler;
+
+    /**
+     * @param \EzSystems\PlatformHttpCacheBundle\PurgeClient\PurgeClientInterface $purgeClient
+     * @param \eZ\Publish\SPI\Persistence\Handler $persistenceHandler
+     */
+    public function __construct(
+        PurgeClientInterface $purgeClient,
+        PersistenceHandler $persistenceHandler
```

Comment:
why do you need to clear location id? you can just clear by content id, and it will clear all relevant cache items. This is one of the many points with multi tagging, you don't need to always refer to cache by location id.